command/server.go

```diff
@@ -1809,6 +1821,49 @@ func (c *ServerCommand) Run(args []string) int {
 				pFile.Close()
 			}
 
+			c.logger.Info(fmt.Sprintf("Wrote pprof files to: %s", pprofPath))
```

Comment:
It feels like this log message would be better moved to inside the case statement, after everything has completed, no?